### PR TITLE
Align with source of production site, exist-db.org

### LINKED ISSUE #3: Align source with deployed version on exist-db.org
The announcements in `index.html` are out of date. This indicates that the currently running version has modifications not present in source control.

### PR DESCRIPTION
**Pull request by joewiz**
- I downloaded exist-db.org/exist/apps/homepage via WebDAV and did a directory diff in oXygen.
- This commit includes: (a) files on the server that were not in source control and (b) changes to files in production that were not yet applied to the source control version.
- I took care not to overwrite portions of HTML heads that the build procedure needed.

Closes #3.

### CODE REVIEW

**Comment by wolfgangmm** [comment]
I think the minimized files may be generated by the grunt build, so could be added to gitignore instead.

**Comment by joewiz** [comment]
Sure. I figured that would be done in a separate PR in response to #2.

File: index.html [hunk 1]
```diff
@@ -8,7 +8,10 @@
         <meta name="description" content="eXist Solutions Long Term Support (LTS) and Service Level Agreement (SLA) order form"/>
         <meta name="keywords" lang="en" content="eXistdb, nosql, database, xml, xquery, xforms, xsl, forms, web, development"/>
         <meta name="keywords" lang="en" content="eXistdb, nosql, database, xml, xquery, xforms, xsl, forms, web, entwicklung"/>
-        <meta name="author" content="eXist Solutions GmbH"/>
```
Comment (ljo) [inline index.html]: I changed this to solutions since we all worked on the design and implementation of the website.

Comment (joewiz) [reply index.html]: Ok, I've applied that change.

**Review by dizzzz — APPROVED** [review]
For me it looks ok

**Pull request merged**